Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -257,6 +269,35 @@ - (void)viewDidLoad {
     self.webView.scrollView.layer.anchorPoint = CGPointMake((isRTL ? 1.0 : 0.0), 0.0);
 
     [self tocUpdateViewLayout];
+
+    @weakify(self)
```

Comment:
:+1: for using `KVOControllerNonRetaining`. FYI you can avoid the `weak/strong` dance by using the `observer` argument passed to the block (which is the same as `self` in this case). If you do this, I also recommend you cast `observer` from `id` to whatever type `self` is for more convenient (and safer) type checking.